knip.json

```diff
@@ -1,6 +1,7 @@
 {
   "$schema": "https://unpkg.com/knip@5/schema.json",
   "ignoreBinaries": ["playwright", "test-e2e"],
+  "ignoreDependencies": ["tailwindcss"],
```

Comment:
we should probably only ignore it for the particular workspace it is used in

(and actually if we specify the entry of that workspace we don't need to ignore it)